shared/common-adapters/input.desktop.js

```diff
@@ -114,7 +114,6 @@ export default class Input extends Component {
           rows={this.props.rows}
           rowsMax={this.props.rowsMax}
           style={{...textStyle, ...globalStyles.flexBoxColumn, ...this.props.textStyle}}
-          textAlign='center'
```

Comment:
I'll delve into it more. I assumed that it would not, because of these lines above

    88: const alignStyle = this.props.style && this.props.style.textAlign ? {textAlign: this.props.style.textAlign} : {textAlign: 'center'}

    106: inputStyle={{...(this.props.small ? {} : {marginTop: 6}), ...inputStyle, ...alignStyle, ...this.props.inputStyle}}

and some quick looks at inputs in the app, but I was waiting on visdiff to confirm.